wind.go

```diff
@@ -314,13 +314,14 @@ func (w *Window) lock1(owner int) {
 	w.owner = owner
 }
 
-// Lock locks every text/clone of w
+// Lock locks every observers/clone of w
```

Comment:
Change back to `text`. This one is arguable but seems more correct that way because the closure is only applicable to `*Text` instances.